README.md

```diff
@@ -23,6 +23,7 @@ You can create RabbitMQ resources:
 3. [Binding](.docs/examples/bindings)
 4. [User](.docs/examples/users)
 5. [Vhost](.docs/examples/vhosts)
+5. [Policy](.docs/examples/policies)
```

Comment:
These links are broken and should be `./docs/examples/policies`

Reply:
https://github.com/rabbitmq/messaging-topology-operator/pull/50 fixed the links, rebasing `main` should pick up those changes, then just the new link for `Policy` would need fixing.